main.py

```diff
@@ -0,0 +1,50 @@
+import os
+import csv
+import gspread
+from oauth2client.service_account import ServiceAccountCredentials
+from dotenv import load_dotenv
+from generate_csv import data
+
+
+
+load_dotenv()
+
+
+from os.path import expanduser
+
+CREDENTIALS_PATH = expanduser("~/Desktop/gspread_workspace/gspreadsheet1.json")
```

Comment:
Why do you need this when you getting your creds from the environment variable below